src/pages/papers/index.tsx

```diff
@@ -14,9 +14,9 @@ const PapersPage: NextPageWithLayout = () => {
       }}
       componentProps={{
         docs: {
-          doomslug: '/files/doomslug.pdf',
-          nightshade: '/files/nightshade.pdf',
-          whitePaperNearProtocol: '/files/the-official-near-white-paper.pdf',
+          doomslug: '/papers/doomslug.pdf',
+          nightshade: '/papers/nightshade.pdf',
+          whitePaperNearProtocol: '/papers/the-official-near-white-paper.pdf',
```

Comment:
Let's remove this totally and use a new link on components side, WDYT?

Reply:
I was thinking to keep these links since they are already known publicly, as a way to avoid breaking existing references to these files.